api_app/script_analyzers/file_analyzers/manalyze.py

```diff
@@ -0,0 +1,28 @@
+from api_app.helpers import get_binary
+from api_app.script_analyzers.classes import FileAnalyzer, DockerBasedAnalyzer
+
+
+class Manalyze(FileAnalyzer, DockerBasedAnalyzer):
+    name: str = "Manalyze"
+    url: str = "http://static_analyzers:4002/manalyze"
+    # interval between http request polling
+    poll_distance: int = 10
+    # http request polling max number of tries
+    max_tries: int = 60
```

Comment:
60 requests is a big number. We should change this to about 10. In your testing, did you find manalyze to take a lot of time processing a binary ?

Reply:
No, It instantly gave the results. But it may take time on bigger binaries. I'm not sure what limit should I keep there.

Reply:
I guess that's fair then. `60` requests is the worst case scenario.